pkg/operator/sync.go

```diff
@@ -906,6 +932,19 @@ func (optr *Operator) applyManifests(config *renderConfig, paths manifestPaths)
 				return err
 			}
 		}
+
+		for _, path := range paths.service {
```

Comment:
interesting, is this how the webhooks are rendered in manifests? as a service? or is this for something else. If so, a comment would be useful here.